src/renderEpisodeButtons.js

```diff
@@ -0,0 +1,14 @@
+import { renderElementIntoDestiny } from './genericFunctions';
+
+const blank = '';
+const episodesPerPage = 20;
+
+export function renderEpisodeButtons(RaMQuery) {
+    const RaMResults = RaMQuery.data.data.episodes.results;
+    renderElementIntoDestiny('episodeButtonList', 'div', blank, 'episodeButtonList', 'episodeBox');
+    for (let index = 0; index < episodesPerPage; index++) {
+        var buttonId = [];
+        buttonId[index] = 'button' + index;   
+    }
+    RaMResults.forEach(episode =>renderElementIntoDestiny(buttonId[episode], 'button', episode.episode, 'episodeButton', 'episodeButtonList'));
```

Comment:
nit: aqui tem uma dica bacana pra facilitar sua vida! oq vc fez pra passar os parametros funciona bem, mas quando ele tem mais que 3 argumentos, fica impossivel decorar oq significa cada "posição" dos argumentos né? O que significa a 4a posição? e a 5a? vc mesmo deve ter ido e voltado na implementação da função pra saber oq era mesmo


Um outro jeito é vc passar um objeto como argumento. Sua função receberia apenas 1 argumento, mas vc pode **NOMEAR** cada chave do objeto:
``` js
function renderElementIntoDesting(argument) {
  const elementId = argument.elementId;
  const elementType = argument.elementType;
  const content = argument.content;
  const elementClass= argument.elementClass;
  const destinyId= argument.destinyId;
  // resto
}
```
Isso fica meio verboso né? Mas vc pode traduzir EXATAMENTE esse comportamentona seguinte sintaxe:
``` js
function renderElementIntoDesting({ elementId, elementType, content, elementClass, destinyId
 }) {
  // resto
}
```

bem limpo né?

e ai pra usar, vc precisaria usar assim:
``` js
RaMResults.forEach(episode =>renderElementIntoDestiny({
  elemntId: buttonId[episode],
  elementTye: 'button', 
   content: episode.episode,
  elementClass: 'episodeButton', 
  destinyId: 'episodeButtonList
}));
```

percebe q eu to passando UM (1) argumento só, criando um objeto SOH pra passar cada argumento? quem estiver lendo sempre vai saber o significado de cada coisa, melhorando a LEGIBILIDADE do código (uma das coisas masi importantes pra considerar enquanto vc coda)